src/dash/admin_view/AdminDash.jsx

```diff
@@ -1,16 +1,168 @@
 import React from 'react';
+import ApproveTutors from './ApproveTutors.jsx';
+import TutorStatistics from './TutorStatistics.jsx';
+import Members from './Members.jsx';
 
-class DefaultDash extends React.Component {
+
+const tutors = [
```

Comment:
Like we did with the student dash, we should add a backend route that will return all tutors that need to be approved